root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -2979,6 +2979,56 @@ const testData = [
             expected_clean_url: 'http://www.qim.com/oeuvres/oeuvre.asp?oeuvreid=716&albumid=16',
        only_valid_entity_types: ['work'],
   },
+  // RateYourMusic
+  {
+                     input_url: 'http://www.rateyourmusic.com/artist/johanna_beyer',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://rateyourmusic.com/artist/johanna_beyer',
+        only_valid_entity_types: ['artist'],
+  },
+  {
+                     input_url: 'https://rateyourmusic.com/concert/riverside_theatre_f1/merzbow_f3',
+             input_entity_type: 'event',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://rateyourmusic.com/concert/riverside_theatre_f1/merzbow_f3',
+        only_valid_entity_types: ['event'],
+  },
+  {
+                     input_url: 'https://rateyourmusic.com/label/tzadik/',
+             input_entity_type: 'label',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://rateyourmusic.com/label/tzadik/',
+        only_valid_entity_types: ['label'],
+  },
+  {
+                     input_url: 'https://rateyourmusic.com/venue/auditorium_parco_della_musica/',
+             input_entity_type: 'place',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://rateyourmusic.com/venue/auditorium_parco_della_musica/',
+        only_valid_entity_types: ['place'],
+  },
+  {
+                     input_url: 'https://rateyourmusic.com/release/single/tori_amos/a_sorta_fairytale/',
+             input_entity_type: 'release',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://rateyourmusic.com/release/single/tori_amos/a_sorta_fairytale/',
+        only_valid_entity_types: ['release'],
+  },
+  {
+                     input_url: 'https://rateyourmusic.com/classifiers/Nonesuch+Explorer+Series/',
+             input_entity_type: 'series',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://rateyourmusic.com/classifiers/Nonesuch+Explorer+Series/',
+        only_valid_entity_types: ['series'],
+  },
+  {
+                     input_url: 'https://rateyourmusic.com/work/funktion_violett/',
+             input_entity_type: 'work',
+    expected_relationship_type: 'otherdatabases',
+            expected_clean_url: 'https://rateyourmusic.com/work/funktion_violett/',
+        only_valid_entity_types: ['work'],
+  },
```

Comment:
Please add validation tests for pages that are being refused intentionally: /~, /film(s), /list, /wiki